src/snac/data/Constellation.php

```diff
@@ -1771,115 +1773,115 @@ public function diff($other, $strict = false) {
             $intersection->setArkID($this->getArk());
         }
 
-        if ($this->getEntityType() != null && $this->getEntityType()->equals($other->getEntityType(), $strict)) {
+        if ($this->getEntityType() != null && $this->getEntityType()->equals($other->getEntityType(), $strict, $checkSubcomponents)) {
```

Comment:
This might be `Term`'s equals, which doesn't have the additional parameter.  So, it might not even need the `$strict` on there either.